masu/config.py

```diff
@@ -46,8 +46,11 @@ class Config(object):
     SQLALCHEMY_TRACK_MODIFICATIONS = False
 
     SECRET_KEY = os.getenv('MASU_SECRET_KEY')
-    if not SECRET_KEY:
+    if SECRET_KEY is None:
```

Comment:
`if not SOME_VAR` is the correct, pythonic way of doing this check.

Under the covers, it calls the `.__nonzero__()` method to check truth, where doing `is None` *only* compares the SECRET_KEY reference to None.